windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -1869,55 +2011,63 @@ function TCompileKeymanWeb.WriteCompiledKeyboard: string; {UTF8}
 
     fkp := fgp.dpKeyArray;
     HasRules := False;
-		for j := 0 to Integer(fgp.cxKeyArray) - 1 do    // I1964
+
+    if IsKeyboardVersion10OrLater or True then
```

Comment:
Why ` or True`?

Reply:
Good catch. I should clean this one up.

Reply:
And fixed. Now only applies the fix is the ladderlength registry value != 0 (default if the registry value is not set is 100).